.gitignore

```diff
@@ -0,0 +1,16 @@
+# Common files and folders
+.vscode/*
+
+# OS generated files
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+# Example build files
+examples/basic/basic
```

Comment:
Not sure where these directories come from? I do not see how that would be created by anything *currently* in the project.

Reply:
This is just in case you use `go build .` instead of `go run`.